### PR TITLE
Ft booking form

### PR DESCRIPTION
**Pull request by ampaire**
I implemented the booking form for the hotels

### CODE REVIEW

File: src/components/views/BookingForm/Form.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+            </div>
+          </div>
+          </form>
+      </div>
+      </div >
+    )
+  }
+}
+
+export default BookingForm;
```
Comment (Austinove) [inline src/components/views/BookingForm/Form.js:197]: It seems like you did not include the forms in the project views.
And i like the way you comment your work.

**Review by Austinove — CHANGES_REQUESTED** [review]
well done, but i could't see the forms

**Review by SaidiAli — CHANGES_REQUESTED** [review]
The PR had conflicts

**Comment by SaidiAli** [comment]
Sorry, but I had to close this.

**Pull request closed without merging**